### PR TITLE
Remove unused commands, scripts

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/validate-generated-code.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2
     - name: Generate code
-      uses: docker://gcr.io/pipecd/codegen:0.3.0
+      uses: docker://gcr.io/pipecd/codegen@sha256:891f26a18497b6d4a3718cc41d109e99a019a7911fe556d281f24d2c7d6eb127 #v0.5.0
```
Comment (knanao) [inline .github/workflows/validate-generated-code.yaml:15]: Could you tell me the reason why use sha instead of tag version?

**Review by knanao — COMMENTED** [review]
Nice catch!

File: .github/workflows/validate-generated-code.yaml (hunk 1 above)
Comment (nghialv) [reply .github/workflows/validate-generated-code.yaml:15]: This docker image is our one so we may don't need to do this.
But I want to ensure exactly what we are using. Even that tag was overridden the old one will be used.

And as GitHub Actions doc:

> Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload.

Comment (knanao) [reply .github/workflows/validate-generated-code.yaml:15]: Oh, I see. That's exactly 'Be Secure'!
Thank you.

**Comment by khanhtc1202** [comment]
Here you go
/approve

**Pull request merged**